### PR TITLE
Tickets/DM-39956

### PR DESCRIPTION
**Pull request by suberlak**

### CODE REVIEW

File: index.ipynb [hunk 1]
```diff
@@ -0,0 +1,283 @@
+{
```
Comment (jbkalmbach) [inline index.ipynb:1]: <p>Simulating <strong>a </strong>grid of stars?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='3'/>

Comment (jbkalmbach) [inline index.ipynb:1]: <ul><li>It's kind of hard to follow the different settings in the opening paragraph. Could you add a table describing the different simulations?</li><li>Colorbar on plots. Are they the same scale?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='4'/>

Comment (jbkalmbach) [inline index.ipynb:1]: <p>typo " sourcve"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='5'/>

Comment (jbkalmbach) [inline index.ipynb:1]: <p>typo "conferted"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='8'/>

Comment (jbkalmbach) [inline index.ipynb:1]: <ul><li>Split these two sections up into different cells</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='9'/>

Comment (jbkalmbach) [inline index.ipynb:1]: <ul><li>These plots are really busy and hard to read. Maybe for the first set you could just pick a bright mag and a faint mag instead of all magnitudes. For the last two maybe a 2x2 grid separating out each corner sensor into its own plot?</li><li>Include a couple sentence explaining exactly how you accomplished the averaging vs stacking analysis. I think you told us in person that you created single donut stamps from stacks to get the stacked results right? </li><li>Also it would be interesting to see a sample from one of the experiments of the single stacked Zernike estimate, the average Zernike estimate and the range of individual Zernike estimates that go into the average. </li><li>Furthermore based upon the above, could you comment based upon that plot requested on how robust each method is to a single detected donut with a bad pixel or other feature that may skew the Zernike estimate based upon that stamp?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='10'/>

Comment (jbkalmbach) [inline index.ipynb:1]: <ul><li>typo "thatr"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='12'/>

**Comment by jbkalmbach** [comment]
Can you re-request review after addressing comments?

File: index.ipynb (hunk 1 above)
Comment (suberlak) [reply index.ipynb:1]: <p>added a table and colorbars to each plots.  Previously they were set by <code>ZScaleInterval()</code> and were not identical (although within few counts)  for each intra/extra pairs.  </p>

Comment (suberlak) [reply index.ipynb:1]: <ul><li>chosen for the first set (comparing stacking vs averaging for different corner sensors with no background) only mag 12 (brightest) and 17 (faintest), to show that for no background there's little difference between the two strategies (and removed the previous ones that showed a variety of magnitudes per sensor).</li></ul>

Comment (suberlak) [reply index.ipynb:1]: <ul><li>split the PSF FWHM plot into a 2x2 panel plot with one panel per corner sensor </li></ul>

Comment (suberlak) [reply index.ipynb:1]: <ul><li>added a plot that illustrates the variance of Zernikes that went into the average</li></ul>

Comment (suberlak) [reply index.ipynb:1]: <ul><li>added a plot summarizing standard deviation of individual Zernike mode estimates</li></ul><p> </p>

Comment (suberlak) [reply index.ipynb:1]: <ul><li>added a plot illustrating how donuts arrays were stacked within each group </li></ul>

File: index.ipynb [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
```
Comment (jbkalmbach) [inline index.ipynb:1]: <p>typo "perform&nbsp;<strong>a&nbsp;</strong>comparison"</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/#comment-1332105391'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

Comment (jbkalmbach) [inline index.ipynb:1]: <p>Here and in section title for simulation above, should it be "of&nbsp;<strong>the&nbsp;</strong>OPD"? Also in the text there are times I think it should be <strong>the </strong>OPD?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/#comment-1332105396'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='7'/>

Comment (jbkalmbach) [inline index.ipynb:1]: <p>Still having trouble seeing the results with quick background after this sentence: we see degradation of Zernike estimate departing further from OPD as the source brightness decreases (source magnitude increases)</p><p>Can you make them more like the noBkgrd results above them?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/#comment-1332105398'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='13'/>

**Review by jbkalmbach — APPROVED** [review]
Looks good. Just some grammar and a request on one last plot.

File: index.ipynb (hunk 2 above)
Comment (suberlak) [reply index.ipynb:1]: <p>Added plots for <code>qckBkgnd</code> that only include two magnitudes  - brightest (12) and faintest (17) to more cleanly show the difference from the OPD,  and that the main difference is in the source magnitude used, rather than whether we're performing averaging vs stacking.  </p>
